string move_type

int32 servo_angle

float64 setpoint_x_angle

float64 setpoint_y_angle

int32 max_speed

int32 stop

int32 set_work_zero